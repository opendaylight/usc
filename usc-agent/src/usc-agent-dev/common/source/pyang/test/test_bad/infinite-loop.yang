module infinite-loop {
  namespace "urn:test";
  prefix "infinite";
  
  grouping target {
    container c;
    uses target; // error: cyclic grouping!
  }

}
