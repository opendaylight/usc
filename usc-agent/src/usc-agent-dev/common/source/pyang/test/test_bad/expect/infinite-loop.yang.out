infinite-loop.yang:5: error: CIRCULAR_DEPENDENCY
infinite-loop.yang:7: error: GROUPING_NOT_FOUND
